### ISSUE TITLE
Half the document missing after cleaning

### ISSUE THREAD

**Issue by finartist** [sylvainhalle/textidote#154]
I tried the tool and was very surprised that there was no warning in about half my document. After I looked at the cleaned file, I can see that half of it is missing.

Here is an extract from the file
```
From this, we can obtain $\tvy_\varepsilon$ and therefore calculate $\tvf(\tvy)$. We used a symbolic computing environment to simplify it to the form

\begin{equation}
\tvf(\tvy) = \nmP^T \, \nvv
\label{eq:opt_flow_field}
\end{equation}

where $\nmP$ is the $9 \times 2$ matrix

\begin{equation}
\nmP =
\begin{pmatrix}
  \frac{b}{depth(\tvy)} \left( \frac{1}{a^2} \mA \tmY - \mR_{12} \right)
   \\[10pt]
  \frac{1}{a^2} \mA \tmY - \mR_{12} - \frac{1}{a^2} \mR_{12} (\tvy \, \tvy^T)
  -  \frac{\cot \alpha}{a} \mR_{11} \, \tmY'
  \\[10pt]
  \frac{1}{ c \; \sin \alpha} \mR_{11} \, \tmY'
\end{pmatrix}
\label{eq:PMatrix}
\end{equation}

with

\begin{equation}
  \begin{aligned}
\mA &= (\va, \va) \; ,  & \mR_{12} &= (\vr_1, \vr_2) \; , & \mR_{11} &= (\vr_1, \vr_1) \; , \\
\tmY &= \mbox{diag}(\tvy) \; ,
& \tmY' &=
\mbox{diag} \left(
\begin{pmatrix}
  0 & -1 \\
  1 & 0
\end{pmatrix} \tvy
\right) \; ,
\\
a &= \mnorm{\va} \; , & b &= \mnorm{\va + \mR_{12} \, \tvy} \; , & c &= \mnorm{\vu}
\end{aligned}
\label{eq:PMatrix_add_def}
\end{equation}

and $\alpha$ is the angle between $\vu$ and $\va$. Note that $\nmP$ only depends on $\nvx, \tvy$ and the depth field $depth(\tvy)$. \hl{$\nmP^T$ is the Jacobian matrix of $\tvy_\vp(\nvx)$ (the projection of $\vp$ to the image plane for a camera $\nvx$) so that $\nmP_{ij} = \mpart{\tvy_i}{x_j}$ and therefore $\tvf(\tvy)$ is the directional derivative in the direction of $\nvv$.}
```

And this is what it looks like after the cleaning
```
From this, we can obtain X and therefore calculate X. We used a symbolic computing environment to simplify it to the form


where X is the X matrix
```
The cleaned file just ends there. But as you can see, there would still be text. In fact, there would be a lot of text following. I can see that my `\hl` macro might get stripped but even normal text is missing.

When I let the analysis run, I get an error in the singleline output option but it seems unrelated
```
Main.tex(L47C3-L47C258): A caption should end with a period "  Exception in thread "main" java.lang.StringIndexOutOfBoundsException: begin 2, end 258, length 18
	at java.base/java.lang.String.checkBoundsBeginEnd(String.java:3319)
	at java.base/java.lang.String.substring(String.java:1874)
	at ca.uqac.lif.textidote.render.SinglelineAdviceRenderer.renderExcerpt(SinglelineAdviceRenderer.java:80)
	at ca.uqac.lif.textidote.render.SinglelineAdviceRenderer.render(SinglelineAdviceRenderer.java:57)
	at ca.uqac.lif.textidote.Main.mainLoop(Main.java:656)
	at ca.uqac.lif.textidote.Main.mainLoop(Main.java:126)
	at ca.uqac.lif.textidote.Main.main(Main.java:112)
```

Can someone assist me with this? Did I mess something up in the .tex file so that it ignores half of it?

Edit: Using Ubuntu 20.04, openjdk 11.0.8 2020-07-14 and version 0.8.1 of TeXtidote

**Comment by sylvainhalle** [comment]
The problem are the lines with `\\[10pt]`. Removing these lines restores the expected behavior. I'll see why TeXtidote breaks when it encounters this line.

**Comment by sylvainhalle** [comment]
Just re-ran the latest version of the code with your example file, and this time the `--clean` option outputs everything. Probably some other fix I implemented also fixed this issue. Should be fixed when release 0.8.2 comes out!
